active_campaign_api/active_campaign_api.py

```diff
@@ -181,11 +195,37 @@ def _prepare_path(
             key = urllib.parse.quote(str(key))
             query_string = f'{query_string}{key}={value}&'
 
```

Comment:
```suggestion
            else:
                querry_string=''

```
to make the function a no-op if the `query_params` are empty. I know it does not matter for your code but it would be nice in case someone else uses this helper function somewhere else.